pdf/creator/creator_test.go

```diff
@@ -37,6 +37,10 @@ const testImageFile2 = "./testdata/signature.png"
 const testRobotoRegularTTFFile = "./testdata/roboto/Roboto-Regular.ttf"
 const testRobotoBoldTTFFile = "./testdata/roboto/Roboto-Bold.ttf"
 const testWts11TTFFile = "./testdata/wts11.ttf"
+
+// XXX: /tmp/2_p_multi.pdf which is created in this test gives an error message when opened in
+//      Adobe Reader: The font FreeSans contains bad Widths.
+//      This problem did not occur when I replaced FreeSans.ttf with LiberationSans-Regular.ttf
```

Comment:
Add author github handle for clarification.  For example
```go
// XXX(author): ....
```

Reply:
We should probably do this going forward and add something about it in the style guide.